docs/products/opensearch/howto/opensearch-dashboard-multi_tenancy.rst

```diff
@@ -14,26 +14,26 @@ Prerequisites
 Configure multi-tenancy in OpenSearch® Dashboard
 -------------------------------------------------
 
-This section provides information on configuring multi-tenancy in OpenSearch Dashboard, which involves enabling OpenSearch Security management, creating tenants, assigning roles, and mapping roles to users.
+This section provides information on configuring multi-tenancy in OpenSearch Dashboard, which involves enabling OpenSearch Security management, creating custom tenants, assigning roles, and mapping roles to users.
 
 Step 1: Enable OpenSearch® Security management
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-To set up multi-tenancy in OpenSearch Dashboard, the initial step is to :doc:`enable OpenSearch Security management <../howto/enable-opensearch-security>` on your Aiven for OpenSearch service. This will provide necessary authentication and authorization features required for multi-tenancy. 
-To enable OpenSearch Security management, see enable OpenSearch . 
+
+If you use the default tenants (Private and Global) in OpenSearch Dashboard, you do not need to enable OpenSearch Security management on your Aiven for OpenSearch service. However, if you want to support custom tenants, enabling OpenSearch Security management is required as it provides necessary authentication and authorization features.
```

Comment:
Possibly move this chapter before the step instructions to make it clear that if you are only interested in the private/global tenants and do not need to limit access to the global tenant you do not need to enable the security management?

Reply:
Good point! I will move this content before the step.

Reply:
Thank you, apart from the error triggered by automated check looks ok to me